pycm/pycm_param.py

```diff
@@ -68,6 +68,7 @@
 VECTOR_ONLY_ERROR = "This option only works in vector mode"
 VECTOR_UNIQUE_CLASS_ERROR = "The classes list isn't unique. It contains duplicated labels."
 CLASS_NUMBER_ERROR = "Number of the classes is lower than 2"
+METRICS_OFF_ERROR = "This method doesn't work in `metrics off` mode."
```

Comment:
@alirezazolanvari 
Would you happen to have any suggestions here?

Reply:
My suggestion is:
"This method cannot be executed while `metrics_off = True`. "

Reply:
`metrics_off=True`